rdr_service/data_gen/generators/data_generator.py

```diff
@@ -327,12 +328,16 @@ def _participant_with_defaults(self, **kwargs):
             'hpoId': UNSET_HPO_ID,
             'withdrawalStatus': WithdrawalStatus.NOT_WITHDRAWN,
             'suspensionStatus': SuspensionStatus.NOT_SUSPENDED,
-            'participantOrigin': 'example',
+            'participantOrigin': 'ppsc',
             'version': 1,
             'lastModified': datetime.now(),
             'signUpTime': datetime.now(),
             'isTestParticipant': False
         }
+
+        if self.project == 'test':
+            defaults.update(participantOrigin='example')
+
```

Comment:
There is no need here to add another variable to the constructor, we have env variables for unit test flags. 

```
if os.environ.get('UNITTEST_FLAG') == “1":
             defaults.update(participantOrigin='example
```